pypeit/core/wavecal/waveio.py

```diff
@@ -239,7 +239,7 @@ def load_line_lists(lines, unknown=False, skip=False, all=False, NIST=False,
 
     Parameters
     ----------
-    lamps : list
+    lines : list
```

Comment:
It would be preferable to have lamps refer to a list of arc lamps (e.g. ['Th', 'Ar']), and lines to be a list of lines. I'm not actually sure what lines is supposed to contain in these codes, but I'm imagining it to be a set of wavelengths, whereas lamps is a set of lines. It would be easier to follow these codes if consistent terminology is used.